quote.py

```diff
@@ -3,48 +3,102 @@
 import random
 from halibot import HalModule
 
+sr = random.SystemRandom()
+
 class Quote(HalModule):
 
-	quotes = []
+	# Configuration defaults
+	remove_votes = 3
+	max_history = 10
```

Comment:
Certainly not a problem to do it this way, but I like the verbosity/clarity this approach provides. The redundancy is a slight cost, of course :P